src/components/Video.js

```diff
@@ -6,6 +6,10 @@ export default class Video extends React.Component {
         return (<div className="container container--md align-center">
             <ReactPlayer
                 url="https://www.youtube.com/watch?v=XV5brOMpzJA" />
+            <p />
```

Comment:
is <p /> the common way to add spacing? 
You may use a spacer component or something like that

Reply:
It was the fastest way I found :) Do you have a recommendation?

Reply:
maybe a div with the css class 'spacer' and the css definition of course 
this makes something like this also possible 
`<div class=spacer>< /div>`
`<div class=spacer--20>< /div>`
`<div class=spacer--40>< /div>`
for different sizes 
but not really necessary now

Reply:
I changed it to <br />. Should be OK for the moment.